### PR TITLE
:running: Enable the lint check goimports 

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description**
Enable the lint check goimports
Note that nothing needed to be fixed in this case.

### CODE REVIEW

File: scripts/verify.sh [hunk 1]
```diff
@@ -38,15 +38,15 @@ golangci-lint run --disable-all \
...
 ##todo(camilamacedo86): The following checks requires fixes in the code
 # --enable=golint
 # --enable=gocyclo
-# --enable=goimports
 # --enable=lll
 # --enable=goconst
 # --enable=gosec
 # --enable=maligned
 
-GO111MODULES=on go list -mod=readonly ./...
```
Comment (estroz) [inline scripts/verify.sh]: Does `golangci-lint run --enable=goimports` supersede this check?

Comment (camilamacedo86) [reply scripts/verify.sh]: You check to fast before my fix pr :-) 
Tks for the help .

File: testdata/project-v2/Makefile [hunk 2]
```diff
@@ -52,7 +52,7 @@ vet:
 
 # Generate code
 generate: controller-gen
-	$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate.go.txt paths="./..."
+	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
```
Comment (estroz) [inline testdata/project-v2/Makefile]: Is this change related to enabling the goimports linter?

Comment (camilamacedo86) [reply testdata/project-v2/Makefile]: To do the PR's we need to run `make generate` which will update the testdata.
Some PR was merged in the master without this update. (may the PR passed in the CI and was merged after the change which affects this artefact)

So, until it is merged (ps.: has a PR already for that  + many ones :-P ) you will see that in all diffs since whiteout it is not possible to check the CI. Mainly for we allow do the reviews.

Comment (Adirio) [reply testdata/project-v2/Makefile]: #1288 fixes it in a separate PR and is tracking all the affected PRs.

**Comment by Adirio** [comment]
LGTM after #1288 gets merged. Directly related to #1183 and #1187.
